package.json

```diff
@@ -33,68 +34,64 @@
     "credential-plus-pbkdf2": "2.0.4",
     "cuid": "2.1.6",
     "dotenv": "5.0.1",
-    "electron-context-menu": "0.11.0",
-    "electron-debug": "2.1.0",
-    "electron-is-dev": "1.0.1",
-    "electron-log": "3.0.5",
-    "electron-settings": "3.2.0",
-    "electron-unhandled": "2.1.0",
-    "electron-updater": "4.0.6",
+    "electron-context-menu": "3.1.1",
+    "electron-debug": "3.2.0",
+    "electron-is-dev": "2.0.0",
+    "electron-log": "4.3.0",
+    "electron-settings": "4.0.2",
+    "electron-unhandled": "3.0.2",
+    "electron-updater": "4.3.0",
     "electron-window-state": "5.0.3",
     "fast-password-entropy": "1.1.1",
     "json-stringify-safe": "5.0.1",
-    "lodash": "4.17.15",
+    "lodash": "4.17.21",
     "metronome-contracts": "2.5.0",
     "metronome-wallet-core": "2.4.9",
```

Comment:
this needs to be updated once  https://github.com/autonomoussoftware/metronome-wallet-core/pull/34 is published